tests/dataset_fields_test.py

```diff
@@ -309,7 +309,7 @@ def test_make_raw_model():
         creationTime=dateutil.parser.parse("2142-04-02T16:44:56"),
         owner="Ponder Stibbons;Mustrum Ridcully",
         ownerGroup="faculty",
-        principalInvestigator="p.stibbons@uu.am",
+        principal_investigator="my principal investigator",
```

Comment:
You got the names here and in line 302 the wrong way around

Reply:
Do you mind explaining? 
I'm confused which one is the model retrieved from SciCat and which one if the scitacean object.
Thanks

Reply:
Look at where the classes are imported from. `RawDataset` comes from `model` and is what goes through the API. `Dataset` comes from `scitacean` directly (strictly speaking `scitacean.dataset`) and is the user facing class with Scitacean's naming scheme.